src/axolotl/utils/data.py

```diff
@@ -160,8 +161,15 @@ def load_tokenized_prepared_datasets(
                         split=None,
                     )
                 elif local_path.is_file():
+                    ds_type = "json"
+                    if d.ds_type:
+                        ds_type = d.ds_type
+                    elif d.data_files and ".parquet" in d.data_files[0]:
```

Comment:
This might error if the user passes `data_files: test.parquet` as string instead of list